content/docs/api-reference/params_show.md

```diff
@@ -0,0 +1,253 @@
+# dvc.api.params_show()
+
+Returns all the <abbr>params</abbr> associated with the given <abbr>DVC
+repository</abbr>.
+
+```py
+def params_show(
+    *targets: str,
+    repo: Optional[str] = None,
+    stages: Optional[Union[str, Iterable[str]]] = None,
+    rev: Optional[str] = None,
+    deps: bool = False,
+) -> Dict:
+```
+
+#### Usage:
+
+```py
+import dvc.api
+
+params = dvc.api.params_show()
+```
+
+## Description
+
+Retrieves the <abbr>params</abbr> tracked in a <abbr>DVC repository</abbr>.
```

Comment:
What if they're in a params file but not used in any stages. Are they still "tracked" ?

Reply:
> What if they're in a params file but not used in any stages. Are they still "tracked" ?

They are not tracked and they won't be retrieved unless the untracked params file is passed as `targets`.

Not sure which alternative could be used to better reflect all the posibilities

Reply:
> unless the untracked params file is passed as targets

- [ ] Good to know, we should add this info to the `targets` arg description.

And don't worry, I'll take this text over once all my doubts have been cleared. Thanks

Reply:
No, that's clear enough. I didn't realize it's already mentioned. Thanks!